tests/test_innit.py

```diff
@@ -46,6 +46,23 @@ def test_ingredients(self):
             self.harvester_class.ingredients(),
         )
 
+    def test_nutrients(self):
+        self.assertEqual(
+            {
+                "sugarContent": "18 g",
+                "proteinContent": "32 g",
+                "fiberContent": "11 g",
+                "unsaturatedFatContent": "55 g",
+                "fatContent": "64 g",
+                "cholesterolContent": "0 mg",
+                "calories": "830 kcal",
+                "carbohydrateContent": "34 g",
+                "saturatedFatContent": "9 g",
+                "sodiumContent": "1060 mg",
```

Comment:
I wonder whether the nutrient fieldnames should be 'abstracted' by `recipe-scrapers`, so that `sodiumContent` here (as supplied by the schema.org definition) becomes a field or method named `sodium`, for example?

The reason for that would be that we may later handle additional embedded web data formats, and they may use slightly different key names when describing these nutritional properties.

It'd be nice if users of the library can call `scrape.nutrients().sodium()` and always get the quantity of sodium from the scraped recipe (if it exists), rather than having to check the `nutrients()` dictionary for `sodiumContent`, `sodium`, `sodiumAmount`, etc - depending on what's retrieved today or in future.

That change could be tracked by opening a new issue and doesn't have to be resolved in this pull request, although I think it'd be good to have a plan in mind before merging.

Reply:
Definitely an idea for new issue/PRs, yes :) Sounds like a mixin is the way to go for what you propose. A separate mixin-type class that will get updates over time and doesn't impact the core functionality